sait/bin/provision_middleware

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+\
```

Comment:
Is this `\` expected?

Reply:
Nice catch... no it was not.

Here's an example of a script file w/out a suffix but with a "#!/bin/bash" 1st line... I obviously did that one manually... and fumble fingered... and didn't test this particular script.